### PR TITLE
refactor: modify the code with standard format.

### PR DESCRIPTION
**Pull request by Aria622**
Modify the code in strategyController.js, using the standard format.

### CODE REVIEW

File: app/controllers/strategyController.js [hunk 1]
```diff
@@ -5,40 +5,21 @@ const { STATUS_TYPE } = require('../utils/statusCodes');
 class StrategyController {
 
   async getAllStrategies(req, res) {
-    try {
-      const strategy = await StrategyService.getAllStrategies();
```
Comment (HuijunDev) [inline app/controllers/strategyController.js:9]: const strategies

**Review by HuijunDev — COMMENTED** [review]
Please change strategy to strategies (getAllStrategies function)

**Review by KevinYang-github — COMMENTED** [review]
I think it's ok

**Review by KevinYang-github — APPROVED** [review]
checked

**Review by HuijunDev — APPROVED** [review]
Done.

**Review by ucfyao — APPROVED** [review]
Excellent

**Pull request merged**